external/common_functions.sh

```diff
@@ -219,6 +226,23 @@ function set_test_info() {
         ubi_packages="git wget unzip"
         ubi_minimal_packages="${ubi_packages}"
         ;;
+    external_custom)
```

Comment:
Does it complicate things to put this content into a test.properties file in the external_custom dir and read from there?  for now we can leave as is, but ultimately the set_test_info() function would be removed completely, and we would read_test_info from each test.properties file found test subdir.

Reply:
I will add a test.properties file in the external_custom dir and add all the contents there for now. After all the variables have been set up and working, the set_test_info() would be removed. I think this should work.